scripts/metricsLib/metrics_definitions.py

```diff
@@ -95,9 +96,27 @@
 SIMPLE_METRICS.append(GraphQLMetric("githubGraphqlSimpleCounts", ["repo", "owner"],
  REPO_GITHUB_GRAPHQL_QUERY, github_graphql_simple_counts_metric_map, token=TOKEN))
 
-PERIODIC_METRICS.append(RangeMetric("newContributorsofCommits", ["repo_id", "period", "begin_date", "end_date"],
-  "https://ai.chaoss.io/api/unstable/repos/{repo_id}/pull-requests-merge-contributor-new?period={period}&begin_date={begin_date}&end_date={end_date}",
-  {"new_commit_contributor": "count"}))
+ORG_METRICS.append(ListMetric("topCommitters", ["repo_group_id"],
```

Comment:
Unrelated but along with org level, are we planning on having a Top Committers metric on a project level too?

Reply:
Yes it would make sense to have it on a repo level as well. I am pretty sure an endpoint for that exists on augur but if it doesn't I can make a PR for it into Augur.

Reply:
Bet sounds good 🙌